backend/src/api/model/event.rs

```diff
@@ -519,6 +519,50 @@ impl AuthorizedEvent {
         Ok(event)
     }
 
+    pub(crate) async fn create_placeholder(event: NewEvent, context: &Context) -> ApiResult<Self> {
+        let query = format!("\
+            insert into all_events ( \
+                opencast_id, title, description, \
+                creators, series, tracks, \
+                read_roles, write_roles, preview_roles, \
+                metadata, is_live, updated, created, state \
+            ) values ( \
+                $1, $2, $3, $4, $5, $6, $7, $8, \
+                '{{}}', '{{}}', false, '-infinity', now(), 'waiting' \
+            ) returning id \
+        ");
+
+        let acl = convert_acl_input(event.acl);
+        let dummy_tracks = vec![EventTrack {
+            uri: "https://example.org/video.mp4".to_string(),
+            flavor: "presenter/preview".to_string(),
+            mimetype: Some("video/mp4".to_string()),
+            resolution: Some([1280, 720]),
+            is_master: Some(true),
+        }];
```

Comment:
Is that really necessary? Looking at `05-events.sql`, it seems like having a track is only required for `ready` events.